bin/vyper

```diff
@@ -21,7 +21,9 @@ format_options_help = """Format to print, one or more of:
  bytecode (default) - Deployable bytecode
  bytecode_runtime   - Bytecode at runtime
  abi                - ABI in JSON format
- python_abi         - ABI in python format
+ abi_python         - Identical to --abi
```

Comment:
Not sure this is identical to `--abi` ?

Reply:
Ah right so they _used to be_. str(o) basically.
```python
        translate_map = {
            'abi_python': 'abi',
            'json': 'abi'
        }
        formats = []
        orig_args = uniq(args.format.split(','))
        for f in orig_args:
            formats.append(translate_map.get(f, f))

        out_list = vyper.compile_codes(codes, formats, output_type='list', exc_handler=exc_handler)
        for out in out_list:
            for f in orig_args:
                o = out[translate_map.get(f, f)]
                if f in ('abi', 'json'):
                    print(json.dumps(o))
                elif f == 'abi_python':
                    print(o)
                elif f == 'source_map':
                    print(json.dumps(o))
                else:
                    print(o)
```
^ this should fix it?

Reply:
That works - 7a893525c9463cf815c0d22f18c216b09b78fcf3